examples/tensorflow2_mnist_ray.py

```diff
@@ -0,0 +1,93 @@
+import tensorflow as tf
+import horovod.tensorflow.keras as hvd
+import horovod.keras as hvd
+
+import ray
+from horovod.ray import RayExecutor
+
+
+def train(num_epochs):
```

Comment:
What kind of function pickling does Ray use?  Is it similar to `cloudpickle` where we can capture variables / imports from outside the function definition?

Reply:
Yep; it is cloudpickle.